lib/data_distributor/cpr/person.rb

```diff
@@ -0,0 +1,83 @@
+module DataDistributor
+  module CPR
+    #This class represents a person as known by cpr
+    class Person
+      attr_reader :data, :cpr
+
+      # @param [Dataset]
+      def initialize(data, cpr:)
+        @data = data
+        @cpr = cpr
+      end
+
+      # Person's first name(s)
+      def first_name
+        data[:Person][:Navn][:fornavne]
+      end
+
+      # Person's last name
+      def last_name
+        data[:Person][:Navn][:efternavn]
+      end
+
+      # returns a Date object with the full date of birth, including century, which is not apparent in the cpr-number
```

Comment:
```suggestion
      # @return [Date] object with the full date of birth, including century, which is not apparent in the cpr-number
```
God kommentar der tilføjer det ekstra information som der er brug for